code-of-conduct.md

```diff
@@ -0,0 +1,165 @@
+# Adafruit CircuitPython Community Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level or type of
+experience, education, socio-economic status, nationality, personal appearance,
+race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+We are committed to providing a friendly, safe and welcoming environment for
+all.
+
+On the Adafruit Discord server, please avoid using nicknames or avatar images
+that might detract from a friendly, safe and welcoming environment. This
+includes but is not limited to use of overtly sexual, vulgar, or otherwise
+offensive nicknames or avatar images.
+
+Please avoid excessive helping. Excessive helping is defined as providing help
+that was not asked for. If someone asks a question, be sure that you're
+providing the help that was requested and not providing unnecessary information
+beyond the scope of the question. Allow the person asking the question to drive
+the conversation. Ask the person to clarify if you're unsure what they are
+asking, instead of providing an answer that covers the question *you think*
+they're trying to ask. If you feel the need to "take the keyboard away" from
+the person requesting assistance, reconsider how you're trying to answer. If
+you're doing all the talking, you may be excessively helping.
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Be kind and courteous to others
+* Behaving professionally
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* The use of inappropriate images in a user's avatar or inappropriate nicknames
+* Any spamming, flaming, baiting or other attention-stealing behavior
+* Excessive helping
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```

Comment:
Recommend "professional or public setting" - As @tannewt mentions in his review, not everyone has been in a professional setting (IE, students/kids). And not every "professional" setting is the same, especially those where children are far removed.